client/styles/app.css

```diff
@@ -1,24 +1,49 @@
 body {
   background-color: lightblue;
 }
+.content {
+  margin: 0.5rem;
+}
 .formular {
-  width: 250px;
+  width: 30rem;
 }
-.content {
-  margin: 2%;
+p {
+  color: red;
```

Comment:
don't use "red" "blue" etc. use hex codes or, as you did well below, the rgb function. also, dont create color palets like a childrens birthday